.gitignore

```diff
@@ -63,3 +63,6 @@ profile.tmp
 .DS_Store
 
 .vscode
+
+# Coverage test
+coverage.txt
```

Comment:
we can probably split this off into its own commit, e.g.
 * `gitignore: add coverage.txt`
 * ` multi: fix typos`

Reply:
fixed